attributes/default.rb

```diff
@@ -45,8 +45,8 @@
 default['hops']['hdfs']['blocksize']           = "134217728"
 default['hops']['hdfs']['umask']               = "0022"
 
-default['hops']['url']['primary']              = node['download_url'] + "/hops-" + node['hops']['version'] + ".tgz"
-default['hops']['url']['secondary']            = "https://www.hops.site/hops-" + node['hops']['version'] + ".tgz"
+default['hops']['download_url']                = node['download_url']
+default['hops']['url']['primary']              = node['hops']['download_url'] + "/hops-" + node['hops']['version'] + ".tgz"
```

Comment:
Generally the PR looks good. I'd just remove the `primary` and call the attribute something like `default['hops']['dist_url']` and change the name of the root url to something like `default['hops']['root_url']`. I suspect that with the current `download_url` people will get confused.

Reply:
change addressed

Reply:
one more thing I need to do